src/Components/Aspire.Azure.AI.Inference/Aspire.Azure.AI.Inference.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
+    <!-- This project needs to multi-target to net9.0 as well to avoid hitting NuGet Restore issues when package is restored from a net9+ project and avoid NU1605 downgrade errors. -->
+    <TargetFrameworks>$(AllTargetFrameworks)</TargetFrameworks>
```

Comment:
Why does this one need it? I'm not seeing a problem.

Reply:
Ah - I was using a `<Project Sdk="Microsoft.NET.Sdk.Web">` app, which automatically lifts `Microsoft.Extensions.DependencyInjection.Abstractions` to 9 because of the aspnetcore shared framework. So that's why I wasn't seeing it. But using a regular console app, it repros.